Post development

aclinton13 opened the pull request.

aclinton13 committed:
Created Server2.java and created a @WebServelt annotation and  hard coded Hello World in doGet. Made a few changes to HelloWorldServerlet.java, but reset it back to original again.

aclinton13 committed:
I added an element to fetch the content in index.html. I created script2.js containing fetch from Server2. Created Server2.java containing the hard coded response from the doGet

aclinton13 committed:
Converted ArrayList to JSON using GSON in Server2. Added dependencies to pom.xml. Created a Json function that returns an object in scrip2.js.

aclinton13 committed:
edited index.html including a form section for users to enter contact information. Added a new webSe
rvelet /form-handler. Contains the names of the input in index.html. Printed it to the server logs. Rendered the input so the user can see it too.

tgoro commented on portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:
```diff
@@ -0,0 +1,33 @@
+    String email = request.getParameter("email");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("firstname: " + firstName);
```
Generally, you wouldn’t submit debugging logs to your final product. I’m not sure how frontend works though and if these are okay style there. I would just Google it!

aclinton13 replied:
Noted, when I was coding it I was thinking the same thing. It seemed to me that they are pretty much accomplishing the same thing.

aclinton13 committed:
Update FormHandlerServlet.java

Removed the debugging logs because they accomplished the same thing as the response section below.
